src/mixins/embed.js

```diff
@@ -106,10 +109,21 @@ export default {
         civicLikerSince,
         amount,
       };
-    }).catch((err) => {
+    }
+    const data = await apiGetDataMinByIscnId(iscnId).catch((err) => {
       console.error(err); // eslint-disable-line no-console
       error({ statusCode: 404, message: '' });
     });
+    const metadata = data && data.data.records[0].data.contentMetadata;
+    const stakeholders = data && data.data.records[0].data.stakeholders;
+    return {
+      id,
+      displayName: stakeholders && stakeholders[0] && stakeholders[0].entity.name,
+      iscnId,
+      amount,
+      avatar: `https://avatars.dicebear.com/api/identicon/${iscnId}.svg`,
```

Comment:
Please add a comment here that we will want to use our own generative art in the future